holoviews/plotting/bokeh/raster.py

```diff
@@ -156,10 +167,404 @@ def get_data(self, element, ranges, style):
+
+    yticks = param.Parameter(default=4, doc="""
+        Ticks along y-axis/annulars specified as an integer, explicit list of
+        ticks or function. If `None`, no ticks are shown.""")
+
+    yrotation = param.Number(default=90, doc="""
+        Define angle along which yticks/annulars are shown. By default, yticks
+        are drawn like a regular y-axis.""")
+
+    # Map each glyph to a style group
+    _style_groups = {'annular_wedge': 'annular',
+                     'text': 'ticks',
+                     'multi_line': 'xmarks',
+                     'arc': 'ymarks'}
+
+    _draw_order = ['annular_wedge', 'multi_line', 'arc', 'text']
+
+    style_opts = (['xmarks_' + p for p in line_properties] + \
+                  ['ymarks_' + p for p in line_properties] + \
+                  ['annular_' + p for p in fill_properties + line_properties] + \
+                  ['ticks_' + p for p in text_properties] + ['width', 'cmap'])
+
+    def __init__(self, *args, **kwargs):
+        super(RadialHeatMapPlot, self).__init__(*args, **kwargs)
+        self.xaxis = None
+        self.yaxis = None
+
+
+    def _get_bins(self, kind, order, reverse=False):
+        """
+        Map elements from given `order` array to bins of start and end values
+        for radius or angle dimension.
+        """
+
+        if kind == "radius":
+            start = self.max_radius * self.padding_inner
+            end = self.max_radius
+
+        elif kind == "angle":
+            start = self.start_angle
+            end = self.start_angle + 2 * np.pi
+
+        bounds = np.linspace(start, end, len(order) + 1)
+        bins = np.vstack([bounds[:-1], bounds[1:]]).T
+
+        if reverse:
+            bins = bins[::-1]
+
+        return dict(zip(order, bins))
+
+
+    @staticmethod
+    def _get_bounds(mapper, values):
+        """
+        Extract first and second value from tuples of mapped bins.
+        """
+
+        array = np.array([mapper.get(x) for x in values])
+
+        return array[:, 0], array[:, 1]
+
+
+    def _postprocess_hover(self, renderer, source):
+        """
+        Limit hover tool to annular wedges only.
+        """
+
+        if isinstance(renderer.glyph, AnnularWedge):
+            super(RadialHeatMapPlot, self)._postprocess_hover(renderer, source)
+
+
+    def get_extents(self, view, ranges):
+        """Supply custom, static extents because radial heatmaps always have
+        the same boundaries.
+
+        """
+
+        lower = -self.padding_outer
+        upper = 2 * self.max_radius + self.padding_outer
+        return (lower, lower, upper, upper)
+
+
+    def _axis_properties(self, *args, **kwargs):
+        """Overwrite default axis properties handling due to clashing
+        categorical input and numerical output axes.
+
+        Axis properties are handled separately for radial heatmaps because of
+        missing radial axes in bokeh.
+
+        """
+
+        return {}
+
+
```

Comment:
There is a fair bit of wasted vertical space in the methods above (extra newline at the end of the docstring and one before the code starts).